speech/google/cloud/gapic/speech/v1/speech_client_config.json

```diff
@@ -6,9 +6,7 @@
           "DEADLINE_EXCEEDED",
           "UNAVAILABLE"
         ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+        "non_idempotent": []
```

Comment:
@shinfan Any idea if this was an intentional change? As in, we _do not_ want to retry `UNAVAILABLE` in Speech?